packages/vue/src/examples/pages/detailed-cart/_internal/OrderSummary.vue

```diff
@@ -34,8 +34,13 @@
         @click="$emit('click:next')"
```

Comment:
There are different text on buttons instead "Place my order" should be "Confirm and pay" like on design. The same on other checkout steps.